app/views/layouts/_branding.html.erb

```diff
@@ -8,25 +8,31 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <% if user_signed_in? && !current_user.org.nil? && current_user.org.logo.present? %>
-        <%= link_to(image_tag(current_user.org.logo.thumb('100x100%').url,
-                              alt: current_user.org.name,
-                              class: "org-logo",
-                              title: current_user.org.name),
-                    current_user.org.target_url) %>
+      <% if user_signed_in? && !current_user.org.nil? %>
+        <% if current_user.org.logo.present? %>
+          <%= link_to(image_tag(current_user.org.logo.thumb('100x100%').url,
+                                alt: current_user.org.name,
+                                class: "org-logo",
+                                title: current_user.org.name),
+                      current_user.org.target_url) %>
+        <% else %>
+          <h4 id="banner-org-name"><%= current_user.org.name %></h4>
+        <% end %>
       <% end %>
     </div>
 
     <!-- Collect the nav links, forms, and other content for toggling -->
     <div class="collapse navbar-collapse" id="org-navbar-menu">
-      <% if user_signed_in? && !current_user.org.nil? %>
-          <ul class="nav navbar-nav">
-            <% current_user.org.links.each do |url| %>
-              <% unless url['link'].blank? %>
-                <li><%= link_to (url['text'].blank? ? url['link'] : url['text']), url['link'] %>
+      <% if user_signed_in? && !current_user.org.nil? && current_user.org.links.present? %>
+        <% if current_user.org.links['org'].present? %>
+            <ul class="nav navbar-nav">
+              <% current_user.org.links['org'].each do |url| %>
+                <% unless url['link'].blank? %>
```

Comment:
We might need to update the concern json_link_validators.rb to prevent blank string but meanwhile this check is good to have, thanks!